### PR TITLE
Rabin2 Xtr: Add error message if none of the plugins can extract bins from the file.

### LINKED ISSUE #4377: rabin2 -x /etc/services no error
it does nothing and it shows nothing

### PR DESCRIPTION
**Pull request by lionaneesh**
Fix #4377

### CODE REVIEW

File: binr/rabin2/rabin2.c [hunk 1]
```diff
@@ -937,6 +937,8 @@ int main(int argc, char **argv) {
 				break;
 			}
 		}
+		// if we reach here, no supported xtr plugins found
```
Comment (radare) [inline binr/rabin2/rabin2.c]: this message will be shown even if the file is extracted properly. the break will only quit the r_list_foreach block, not the whole function

**Comment by lionaneesh** [comment]
@radare Updated!

File: binr/rabin2/rabin2.c [hunk 2]
```diff
@@ -930,13 +930,21 @@ int main(int argc, char **argv) {
 	if (action & ACTION_EXTRACT) {
 		RListIter *iter;
 		RBinXtrPlugin *xtr;
+		bool supported = 0;
```
Comment (radare) [inline binr/rabin2/rabin2.c]: if type is `bool` use `true` and `false` instead of `1` and `0`

**Comment by radare** [comment]
Fixed and merged. Thanks! it looks good now

**Pull request closed without merging**